### PR TITLE
Replace hardcoded values for Pi

### PR DESCRIPTION
**Pull request by wolterhv**
**Description:** Replaces re-definitions of π for their equivalents in `math` (for python) or `math.h` (for C++). See discussion in https://github.com/commaai/openpilot/issues/20072.

**Verification:** All automatic checks passed.

### CODE REVIEW

File: selfdrive/car/ford/carcontroller.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from selfdrive.car import make_can_msg
 from selfdrive.car.ford.fordcan import create_steer_command, create_lkas_ui, spam_cancel_button
 from opendbc.can.packer import CANPacker
+import math
```
Comment (adeebshihadeh) [inline selfdrive/car/ford/carcontroller.py:5]: standard library imports should be at the top

File: selfdrive/car/mock/interface.py [hunk 2]
```diff
@@ -5,12 +5,13 @@
 import cereal.messaging as messaging
 from selfdrive.car import gen_empty_fingerprint
 from selfdrive.car.interfaces import CarInterfaceBase
+import math
```
Comment (adeebshihadeh) [inline selfdrive/car/mock/interface.py:8]: same

File: selfdrive/controls/lib/lateral_mpc/generator.cpp [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #include <acado_code_generation.hpp>
 #include "common/modeldata.h"
+#include <math.h> // M_PI
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/lateral_mpc/generator.cpp:3]: remove the comment

**Comment by wolterhv** [comment]
Thanks for your comments and for fixing the title!

**Pull request merged**